### PR TITLE
Update facia-pressing docs

### PR DESCRIPTION
**Pull request by gtrufitt**
I think Facia Pressing could do with some more automated or single button clicky steps but, for the moment, I've updated the required steps. 

@guardian/dotcom-platform @janua @johnduffell

### CODE REVIEW

File: docs/03-dev-howtos/06-repressing.md [hunk 1]
```diff
@@ -1,61 +1,105 @@
...
 
 #### Prepare the ground
-* choose a quiet time of day, and this could take an hour
-* if it's on, turn off auto deploy of preview (and CODE facia if you like)
-* goo deploy block
-* let the team know what you're doing
-* merge to master
+
+* Choose a quiet time of day, and this will take an hour
+* Can you `ssh` onto the facia-press instances for `CODE` and `PROD`?
```
Comment (johnduffell) [inline docs/03-dev-howtos/06-repressing.md]: you could just link to a doc in platform that explains it and just keep this as one step

File: docs/03-dev-howtos/06-repressing.md [hunk 2]
```diff
@@ -1,61 +1,105 @@
...
+* Set the `facia-press` AWS auto scaling group desired value to 2
 
 #### CODE: get the field into the pressed json
-* deploy facia-press, turn on the facia-press-on-demand switch
-* log into a facia presser and post to http://localhost:9000/press/draft/all (takes a couple of mins) to repress all the draft fronts (curl -v -X POST -H 'Content-Length: 0' http://localhost:9000/press/draft/all)
-* go to frontend admin /press and click press all, then wait a while for all the live fronts to press (you can monitor the SQS queue size to know when it's done - it may be worth scaling the group to 2 while it's happening)
-* check that code facia still works (e.g. /uk) (if you want to debug, go to a facia press load balancer: http://<frontend-faciapre-elb>/pressed/live/uk/money)
+
+* Deploy facia-press & turn on the `facia-press-on-demand` switch in CODE gutools
+* SHH into the facia presser instance and post to http://localhost:9000/press/draft/all (have patience, it takes a couple of mins) to repress all the draft fronts (`curl -v -X POST -H 'Content-Length: 0' http://localhost:9000/press/draft/all`)
```
Comment (johnduffell) [inline docs/03-dev-howtos/06-repressing.md]: ssh

File: docs/03-dev-howtos/06-repressing.md [hunk 3]
```diff
@@ -1,61 +1,105 @@
...
+- `FaciaControllerTest.should render fronts with content that has been pre-fetched from facia-press` 
+
+This is because the pressed JSON does not match what `Facia` is expecting. [Read the Fronts architecture here](https://github.com/guardian/frontend/blob/d4422b4537165424e70a898d150db4e806ba04d6/docs/02-architecture/02-fronts-architecture.md).
+
+### Repressing fronts locally.
+
+#### Prerequisites
+
+* Check your `facia.stage` property in your `frontend.conf` file is pointing at DEV (*not* PROD!)
+* Check you are using your own switches file and not the prod or code one.
```
Comment (jfsoul) [inline docs/03-dev-howtos/06-repressing.md]: CODE/PROD for consistency?

File: docs/03-dev-howtos/06-repressing.md [hunk 4]
```diff
@@ -1,61 +1,105 @@
...
+#### Lets go.
+
+* Delete the files in `data/pressedPage/`
+* Run the `admin` project locally and turn on `facia-press-on-demand` *and* `facia-inline-embeds` [switch](http://localhost:9000/dev/switchboard#Facia)
+* Stop admin and run `facia-press` locally
+* Go to `http://localhost:9000` just to get the switches file warmed up, then
+	* curl -v -X POST http://localhost:9000/press/live/music
+	* curl -v -X POST http://localhost:9000/press/live/inline-embeds
+	* curl -v -X POST http://localhost:9000/press/live/uk
+	* curl -v -X POST http://localhost:9000/press/live/au/media
```
Comment (gtrufitt) [inline docs/03-dev-howtos/06-repressing.md]:
```
good idea 
````

File: docs/03-dev-howtos/06-repressing.md [hunk 5]
```diff
@@ -1,61 +1,105 @@
...
+* Deploy everything else and check they still work
+
+#### Important Cleanup
+
+* Turn off the `facia-press-on-demand` switches
+* Turn on auto deploys again
+* Change your local machine back to DEV facia bucket!
+* Unblock deploys
+* Let Central Prod know things are hunky-dory
+* Celebrate! 🎉
```
Comment (jfsoul) [inline docs/03-dev-howtos/06-repressing.md]: Celebrate **wildy**! 😁

**Comment by jfsoul** [comment]
A big improvement! 💯

File: docs/03-dev-howtos/06-repressing.md [hunk 6]
```diff
@@ -1,61 +1,105 @@
...
+* Check your `facia.stage` property in your `frontend.conf` file is pointing at DEV (*not* PROD!)
+* Check you are using your own switches file and not the prod or code one. 
+	* To do this, create your own properties file in the the S3 bucket named `aws-frontend-store`. Add it to your `frontend.conf` like this: `switches.key=DEV/config/switches-nbaltazar.properties`
+* Ensure you have the test AWS SQS queues in your `frontend.conf` properties
+	* Add two properties: `frontpress.sqs.tool_queue_url` and `frontpress.sqs.cron_queue_url`
+	* Set _both_ properties to the value `"https://sqs.eu-west-1.amazonaws.com/<id-here>/Frontend-TEST-page-presser-queue"`
+	* You can get the correct ID by going to SQS in the AWS console and finding the `Frontend-TEST-page-presser-queue` and find the URL field in the details tab
+* Ensure your CAPI endpoint is PROD (`content.api.host` in `frontend.conf`)
+
+#### Lets go.
```
Comment (desbo) [inline docs/03-dev-howtos/06-repressing.md]: let's

Comment (gtrufitt) [inline docs/03-dev-howtos/06-repressing.md]: I got weirded out by this. Let us go. Sounds weird. but yeh. stupid english.

File: docs/03-dev-howtos/06-repressing.md (hunk 2 above)
Comment (gtrufitt) [inline docs/03-dev-howtos/06-repressing.md]: Shush.

**Comment by gtrufitt** [comment]
Thx all, changes pushed.

**Review by NataliaLKB — APPROVED** [review]
Amazing work! 👍

**Comment by johnduffell** [comment]
great work, docs power up! 👍

**Pull request merged**